Find a functionally equivalent code:android.graphics.Paint.setXfermode:COMMENT
Method Modifier: public      
Comment:/**
 * Set or clear the xfermode object.
 * <p />
 * Pass null to clear any previous xfermode.
 * As a convenience, the parameter passed is also returned.
 *
 * @param xfermode May be null. The xfermode to be installed in the paint
 * @return         xfermode
 */

@@ -1,5 +1,5 @@
 {
-    int xfermodeNative = 0;
+    long xfermodeNative = 0;
     if (xfermode != null)
         xfermodeNative = xfermode.native_instance;
     native_setXfermode(mNativePaint, xfermodeNative);

